x-pack/test/api_integration/apis/endpoint/policy.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect/expect.js';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function({ getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const supertest = getService('supertest');
+  describe('Endpoint policy api', () => {
+    describe('GET /api/endpoint/policy-response', () => {
+      before(() => esArchiver.load('endpoint/policy'));
```

Comment:
for the `load` and `unload` - both of these return a `Promise`, thus you should probably use an async function and `await` those to complete. (also noticed this is not being done in other tests (outside of this PT))